src/axom/primal/geometry/Hexahedron.hpp

```diff
@@ -308,28 +308,6 @@ class Hexahedron
     tets[23] = TetrahedronType(hc, m_points[6], m_points[7], fm6);
   }
 
-  /**
-   * \brief Finds the volume of the hexahedron by triangulating the hexahedron
-   *        into 24 tetrahedrons and finding the total volume of the tetrahedrons
-   *
-   * \return The volume of the hexahedron
-   */
-  AXOM_HOST_DEVICE
-  double volume_tet_decomp()
```

Comment:
Thanks for removing this from the class @bmhan12 

Could you please add this function to the `primal_hexahedron` tests to ensure that `Hexahedron::volume()` generates the same results as summing the subvolumes in `Hexahedron::triangulate()`?

I apologize for not being clearer in my PR comments. My point was that, assuming we get the same value, users would never need to call `volume_tet_decomp()`